pkg/kubectl/cmd/proxy.go

```diff
@@ -83,14 +86,68 @@ func NewCmdProxy(f *cmdutil.Factory, out io.Writer) *cobra.Command {
 	cmd.Flags().StringP("address", "", "127.0.0.1", "The IP address on which to serve on.")
 	cmd.Flags().Bool("disable-filter", false, "If true, disable request filtering in the proxy. This is dangerous, and can leave you vulnerable to XSRF attacks, when used with an accessible port.")
 	cmd.Flags().StringP("unix-socket", "u", "", "Unix socket on which to run the proxy.")
+	cmd.Flags().StringP("exec", "e", "", "Command to be execured with the proxy address.")
 	return cmd
 }
 
+type executorInterface interface {
+	Exec(argv0 string, argv []string) error
+	LookPath(argv []string) (string, error)
+}
+
+type syscallExecutor struct{}
+
+func (*syscallExecutor) Exec(argv0 string, argv []string) error {
+	cmd := exec.Command(argv0, argv...)
+	cmd.Stdout = os.Stdout
```

Comment:
I think we need to use here the `io.Writer` you are receiving in `NewCmdProxy`. Store it in `syscallExecutor` when you instantiate it and use what's stored. Same with stdin and stderr, [run](https://github.com/kubernetes/kubernetes/blob/master/pkg/kubectl/cmd/run.go#L83) is an example that also takes all three.